.travis.yml

```diff
@@ -24,6 +24,9 @@ cache:
         - $HOME/.npm
 
 before_install:
+    - nvm install 0.12
```

Comment:
This is a great change. It seems to fix our build failures with `svgo`!

Would you mind making a separate pull request with only these changes? So we can merge this ASAP.